app/Http/Controllers/V2/Terrafund/TerrafundClipGeometryController.php

```diff
@@ -2,35 +2,53 @@
 
 namespace App\Http\Controllers\V2\Terrafund;
 
-use App\Helpers\CreateVersionPolygonGeometryHelper;
 use App\Helpers\GeometryHelper;
-use App\Helpers\PolygonGeometryHelper;
+use App\Jobs\FixPolygonOverlapJob;
 use App\Models\V2\Sites\CriteriaSite;
+use App\Models\V2\Sites\Site;
 use App\Models\V2\Sites\SitePolygon;
 use App\Services\PolygonService;
-use App\Services\PythonService;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Log;
 
 class TerrafundClipGeometryController extends TerrafundCreateGeometryController
 {
     public function clipOverlappingPolygonsBySite(string $uuid)
     {
+        $user = Auth::user();
         $polygonUuids = GeometryHelper::getSitePolygonsUuids($uuid)->toArray();
+        $job = new FixPolygonOverlapJob($polygonUuids, $user->id);
+        $jobUUID = $job->getJobUuid();
+        dispatch($job);
+
+        return response()->json(['job_uuid' => $jobUUID], 200);
```

Comment:
Let's enforce consistency on these with a Resource class. Something like `DelayedJobResource` that gets returned from each of these endpoints.

Reply:
done